app/views/admin/form_templates/_form.html.erb

```diff
@@ -3,8 +3,18 @@
   <%= f.input :title %>
 
   <%= f.simple_fields_for :properties do |p| %>
-    <%= p.input :visible, :label => "Show #{p.object.name}" %>
-    <%= p.input :name, :as => :hidden %>
+    <div id="<%= p.object.name %>" class="panel panel-default template-property">
+      <div class="panel-heading">
+        <div class="panel-title">
+          <%= p.object.name %>
+        </div>
+      </div>
+      <div class="panel-body">
+        <%= p.input :visible, :label => "Show" %>
```

Comment:
There might also be aria attributes you can put on the panel header, but I'm not sure if those exist or are as effective as a fieldset with legend